oemof_visio/energy_system_graph.py

```diff
@@ -0,0 +1,279 @@
+import os
+import graphviz
+import oemof.solph
```

Comment:
Something like this might work:
```
from oemof.network import (Bus, Sink, Source, Transformer)
try:
    from oemof.solph import GenericStorage
except ModuleNotFoundError:
    GenericStorage = NoneType
```

Reply:
for me `from oemof.network import Bus` raises a `ImportError: cannot import name 'Bus'`

Reply:
```diff
+ from oemof.network import (Bus, Sink, Source, Transformer)
- from oemof.network.network import (Bus, Sink, Source, Transformer)
```

Reply:
oemof network could be an extra requirement. 

```python
try:
    from oemof.network.network import (Bus, Sink, Source, Transformer)
except:
    raise (ImportError, "You have to install oemof.network to plot a graph...pip install oemof.network <or something like this>")
```

99% will not see this message because solph and network is needed to create an EnergySystem.